packages/abi-coder/src/constants.ts

```diff
@@ -1,6 +1,7 @@
 export const OPTION_CODER_TYPE = 'enum Option';
 export const VEC_CODER_TYPE = 'struct Vec';
 export const BYTES_CODER_TYPE = 'struct Bytes';
+export const STRING_CODER_TYPE = 'struct String';
```

Comment:
```suggestion
export const STD_STRING_CODER_TYPE = 'struct String';
```
May be best to rename this in order to keep the different naming conventions for a `StringCoder` and `StdStringCoder`. Will make the switch statement in the `AbiCoder` a bit clearer as well as the string regex check follows the std string check.